sonar-scm-git-plugin/src/main/java/org/sonarsource/scm/git/GitScmProvider.java

```diff
@@ -193,6 +194,38 @@ public BlameCommand blameCommand() {
     return null;
   }
 
+  /**
+   * This method will override API in SQ 8.4
+   */
+  @CheckForNull
+  public Instant forkDate(String referenceBranchName, Path projectBaseDir) {
```

Comment:
Wouldn't it be more future proof to return commit instead of date? (giving the fact that this is a library)

Reply:
A commit object with an id and a date? good idea

Reply:
That would actually require depending on a unreleased version of SonarQube, so I'll leave it as is.